packages/shared/src/graphql/search.ts

```diff
@@ -127,3 +127,16 @@ export const updateSearchData = (
 
   return updated;
 };
+
+export const searchQueryUrl = 'https://api.daily.dev/search/query';
```

Comment:
We should make this a dynamic URL based on the API you are at probably

Reply:
100% agree. Let's pull the `apiUrl` we have.